File: src/adapter/java/com/my/adapter/UserDTOAdapter.java
package src.adapter.java.com.my.adapter;

import src.adapter.java.com.my.dto.UserDTO;
import src.prototype.java.com.my.db.model.User;

public class UserDTOAdapter {
    UserDTO dto;

    public UserDTOAdapter(User user) {
        dto = new UserDTO();
        dto.setEmail(user.getEmail());
        dto.setUsername(user.getUsername());
    }

    public UserDTO getDto() {
        return dto;
    }
}

File: src/adapter/python/adapter/UserDTOAdapter.py
from src.adapter.python.dto.UserDTO import UserDTO
from src.prototype.python.db.model.User import User


class UserDTOAdapter:
    dto: UserDTO = None

    def __init__(self, user: User):
        self.dto = UserDTO(user.email, user.username)

    def getDTO(self):
        return self.dto
